administrator-manual/en/webmail.rst

```diff
@@ -65,4 +65,16 @@ If you want to open the access from any network: ::
 
   config setprop roundcubemail access public
   signal-event nethserver-roundcubemail-update
+  
+Removing
+========
+
+You can remove Roundcube to add another webmail type (eg: SOGo, WebTop.…) and you can keep all the mail server RPMs.
```

Comment:
I would not say "to add another webmail type (eg: SOGo, WebTop.…)", because all of them can be installed on the same system.